packages/EasyCore/src/Bridge/Laravel/Middleware/TrimStrings.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Bridge\Laravel\Middleware;
+
+use EonX\EasyCore\Helpers\CleanerInterface;
+use Illuminate\Http\Request;
+use Symfony\Component\HttpFoundation\ParameterBag;
+
+final class TrimStrings
+{
+    /**
+     * The attributes that should not be trimmed.
+     *
+     * @var string[]
+     */
+    private $except;
+
+    /**
+     * @var CleanerInterface
+     */
+    private $cleaner;
+
+    /**
+     * @param string[] $except
+     */
+    public function __construct(CleanerInterface $cleaner, array $except = [])
```

Comment:
Now we need to update a service provider to be able to configure those "except" fields. Please, update `EasyCoreServiceProvider`: make it possible to add "except" fields to the following place in config - `easy-core.trim_strings.except`.

Reply:
You also need to update `src/Bridge/Laravel/config/easy-core.php`. Provide an empty array as a default for `easy-core.trim_strings.except`

Reply:
And you also need to configure these things for Symfony bridge: look at the `src/Bridge/Symfony/DependencyInjection` directory.

Reply:
Done.